S2 at2 basic form

FelixLetheren opened the pull request.

FelixLetheren committed:
Added icon on index that links to new fish entry page.

FelixLetheren committed:
Created basic HTML form

FelixLetheren committed:
Corrected an alt on newFish.php

FelixLetheren committed:
added label to aggression selector.

sjr-mayden commented on index.php:
```diff
@@ -21,5 +21,10 @@
 </div>
+<div class="link-container">
+    <a class="new-fish-icon" href="newFish.php">
+        <img src="images/plus-icon.png" alt="Add a new fish"
```
You haven't closed the img tag

sjr-mayden replied:
(Covered by validator comment)

sjr-mayden commented on newFish.php:
```diff
@@ -0,0 +1,39 @@
+<?php
+require_once 'php-files/functions.php'
+?>
+<!DOCTYPE html>
```
Run through https://validator.w3.org/nu/#textarea

9 errors and a warning to fix

sjr-mayden commented on newFish.php:
```diff
@@ -0,0 +1,39 @@
+</div>
+<div class="link-container">
+    <a class="new-fish-icon" href="index.php">
+        <img src="images/return.png" alt="Return to Collection"
```
This looks a lot more like a refresh or undo option than a go back option

FelixLetheren committed:
Closed img tag
ran through validator and resolved errors and warnings
changed return image to a clearer icon.

sjr-mayden commented:
"Closed img tag" doesn't accurately reflect the work done in this commit
